girder/wsgi.py

```diff
@@ -18,14 +18,24 @@
 ###############################################################################
 
 import cherrypy
-from girder.utility import config, server
+import girder
+import logging.handlers
+from girder.utility import server
 
-config.loadConfig()  # Read the config files first, so we can override some values
 cherrypy.config.update({'engine.autoreload.on': False,
                         'environment': 'embedded'})
 cherrypy.config['server'].update({'cherrypy_server': False,
                                   'disable_event_daemon': True})
 
+# TODO The below line can be removed if we do away with girder.logprint
+girder._quiet = True  # This means we won't duplicate messages to stdout/stderr
+_formatter = girder.LogFormatter('[%(asctime)s] %(levelname)s: %(message)s')
+_handler = logging.handlers.StreamHandler()  # Send logs to stderr
+_handler.setFormatter(_formatter)
+girder.logger.addHandler(_handler)
+cherrypy.log.access_log.addHandler(_handler)
+cherrypy.log.error_log.addHandler(_handler)
```

Comment:
Do we know what's being sent to `cherrypy.log.access_log` and `cherrypy.log.error_log`. I think it'd be ideal if could ignore these and prefer `girder.logger` as the one logger to rule them all.

Reply:
Internal stuff from cherrypy, I'm assuming. I'm fine either way, was just copying some of the logic that happens inside the old method.